Tutorial_1b_Moment_calculation.ipynb

```diff
@@ -0,0 +1,877 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    rms_spec_realnoise = mad_std(spec_noise_realistic)
</pre><p>Can this line now be removed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
The line below was wrong: needed to include the `rms_spec_realnoise`

Reply:
Ok got it!